src/common/names.ts

```diff
@@ -259,6 +259,7 @@ export const SNAKE_NAMES: Map<number, string> = new Map<number, string>([
     [7, 'Ruby'],
     [8, 'Bailey'],
     [9, 'Max'],
+    [10, 'Misunderstood Animal who did nothing to you'],
```

Comment:
Please remove this one. Such a long name is going to cause havoc. It's more of a sentence than a name :-)